library/std/src/keyword_docs.rs

```diff
@@ -1987,13 +1991,15 @@ mod type_keyword {}
 ///
 /// ```rust
 /// # #![allow(dead_code)]
+/// #![deny(unsafe_op_in_unsafe_fn)]
 /// /// Dereference the given pointer.
```

Comment:
Recommend leaving a blank line between the module-level attribute and the item-level doc comment. Otherwise this one is easy to misread as an item-level attribute. Keeping `#![…]` vs `#[…]` straight is already hard enough for someone at the stage of learning of just learning about unsafe blocks.

```suggestion
/// #![deny(unsafe_op_in_unsafe_fn)]
///
/// /// Dereference the given pointer.
```